types/pinyin/pinyin-tests.ts

```diff
@@ -0,0 +1,23 @@
+import pinyin = require("pinyin");
+
+pinyin("中心");
+pinyin("中心", {
+    heteronym: true
+});
+pinyin("中心", {
+  heteronym: true,
+  segment: true
+});
+pinyin("中心", {
+  style: pinyin.STYLE_INITIALS,
+  heteronym: true
```

Comment:
[use 4 spaces](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes) and it'd better to enable tslint to enforce checking style.